lib/redirect_static_pages.rb

```diff
@@ -0,0 +1,24 @@
+# This enables us to redirect static pages published by PublishStaticPages
+# to specified URLs.
+
+class RedirectStaticPages
+  def redirect
+    pages.each do |page|
+      PublishingApiRedirectWorker.new.perform(page[:content_id], page[:target_path], "en")
+    rescue GdsApi::HTTPNotFound => e
+      # we can't unpublish something that doesn't exist
+      puts e
+      nil
+    end
+  end
+
+  def pages
+    [
+      {
+        base_path: "/government/announcements",
```

Comment:
is base_path needed?

Reply:
Actually no, it just tells you what the current path is. It's not used, so we could remove it if you don't think this is helpful.